subprojects/testfx-core/src/main/java/org/testfx/service/finder/impl/WindowFinderImpl.java

```diff
@@ -138,7 +140,7 @@ public Window window(Node node) {
 
     @SuppressWarnings("deprecation")
     private List<Window> fetchWindowsInQueue() {
-        List<Window> windows = Lists.newArrayList(Window.impl_getWindows());
+        List<Window> windows = Lists.newArrayList(getWindows());
         return ImmutableList.copyOf(Lists.reverse(windows));
```

Comment:
Remove the reversal here, and let the JDK 8 implementation return it in reverse, and the JDK 9 implementation *not* reversed.

Reply:
Ok, looks like this is another method to add to JavaVersionAdapter.

Also, when I did remove the reversal and ran the tests solo, I believe they worked fine. When I run it in context of an entire gradle build, that's when it fails. So, it might be more than just that going on here.